### PR TITLE
Hero overlapped book cover pattern for #86

### LINKED ISSUE #86: Block Patterns - Hero - Overlapped book cover with links
**Description**

🛠️ The goal of this issue is to track the implementation of the `Hero - Overlapped book cover with links` pattern. 
🧑‍💻 The implementation must follow the [pattern creation guidelines](https://github.com/WordPress/twentytwentyfive/blob/trunk/README.md#pattern-creation-guidelines), and should be assigned to its appropriate category.
🎨 [Figma design](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=3177-1794&t=IqDHO3nwfV13Hcvm-1)

Hero_ Overlapped book cover with links

### PR DESCRIPTION
**Pull request by Arkenon**
**Description**

Re-create Hero-Overlapped Book Cover with Link Pattern. (#86)

**Screenshots**

[screen-capture (2).webm](https://github.com/user-attachments/assets/cf64ce30-109d-4999-96d4-a3e607615a63)

### CODE REVIEW

**Comment by carolinan** [comment]
@Arkenon  @juanfra 
There are two separate pull requests now that adds the image of the book as well as the credits in the readme file.
One of them needs to be merged first for the other pattern to take advantage of the same details .

**Comment by Arkenon** [comment]
> @Arkenon  @juanfra 
> There are two separate pull requests now that adds the image of the book as well as the credits in the readme file.
> One of them needs to be merged first for the other pattern to take advantage of the same details .

@carolinan Would you merge my pull request first?

**Comment by carolinan** [comment]
I would love for the two of you to solve it between you 😆 .
This PR uses the wrong image, it still has the trademark symbol.

**Comment by juanfra** [comment]
Although it is the same book, this and the other patterns use a different image. 

The ones for the other PR use an image with a grey background and a shadow, and Bea clarified that [here](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK?node-id=3177-1786#947554968).

**Comment by carolinan** [comment]
Since these are example images that are intended to be replaced by the user, I would prefer if the number of images in the theme is kept to a minimum. They also increase the size of the theme.

**Comment by carolinan** [comment]
This sounds harsh but the theme itself is not the demo.

**Comment by Arkenon** [comment]
> I would love for the two of you to solve it between you 😆 .
> This PR uses the wrong image, it still has the trademark symbol.

I downloaded the image from this link @juanfra sent me:

https://github.com/user-attachments/files/16871410/book-image.webp.zip

**Comment by Arkenon** [comment]
@juanfra Would you send me book-image without symbol? I will change the image. Then I think the pull request can be merged.. (?).

**Comment by juanfra** [comment]
@Arkenon - Sure thing.

[book-image.webp.zip](https://github.com/user-attachments/files/17011572/book-image.webp.zip)

**Comment by Arkenon** [comment]
I have changed the book image:

bookImagePattern

File: patterns/hero-overlapped-book-cover-with-links.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+ * Description: A hero with an overlapped book cover and links.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:cover {"isUserOverlayColor":true,"minHeight":818,"customGradient":"linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)","contentPosition":"center center","align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|contrast"}}},"spacing":{"padding":{"top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"}}},"textColor":"contrast","layout":{"type":"constrained"}} -->
```
Comment (juanfra) [inline patterns/hero-overlapped-book-cover-with-links.php]: Is there any reason to have a minimum height?

File: patterns/hero-overlapped-book-cover-with-links.php [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+<div class="wp-block-cover alignfull has-contrast-color has-text-color has-link-color" style="padding-top:var(--wp--preset--spacing--80);padding-bottom:var(--wp--preset--spacing--80);min-height:818px">
+	<span aria-hidden="true" class="wp-block-cover__background has-background-dim-100 has-background-dim has-background-gradient" style="background:linear-gradient(180deg,rgb(251,250,243) 51%,rgb(255,255,255) 51%)"></span>
+	<div class="wp-block-cover__inner-container">
+		<!-- wp:group {"align":"full","layout":{"type":"constrained","justifyContent":"center","contentSize":"1124px"}} -->
```
Comment (juanfra) [inline patterns/hero-overlapped-book-cover-with-links.php]: Not sure if we should set the content size for this one, I'll defer to @beafialho

File: readme.txt [hunk 3]
```diff
@@ -244,8 +244,13 @@ Free public domain CC0 image.
 link-in-bio-image
 https://www.rawpixel.com/image/14262995
 
+Book Image - Created by Beatriz Fialho (@beafialho) using an image from rawpixel:
+Free public domain CC0 image.
+book-image.webp
+https://www.rawpixel.com/image/8799536
```
Comment (juanfra) [inline readme.txt]: This won't be necessary as we have it two times now. Let's move `book-image.webp` below `book-image-landing.webp`

**Review by juanfra — COMMENTED** [review]
Thank you for working on this @Arkenon - It is looking good to me. I left a few comments and questions for you.

I'll defer to @beafialho, for her to confirm that it looks as expected.

**Comment by beafialho** [comment]
Thank you for working on this one!

I noticed a few things that can be polished:

- Please set the image, which is looking bad, to `align: none` like the screen recording:

https://github.com/user-attachments/assets/59d6b1e6-2cb4-413e-9827-9d83f85e207d

The pattern outer spacing on mobile is a bit too large. To fix this:
- Untoggle the `Inner blocks use content width` in the group inside the cover
- Add `Regular` spacing preset to the cover
- Set the group inside it to wide width instead
- Notice how the spacing looks more precise

https://github.com/user-attachments/assets/272dba58-ec89-4640-bc8e-d5212d8cf3d2

File: patterns/hero-overlapped-book-cover-with-links.php (hunk 1 above)
Comment (Arkenon) [reply patterns/hero-overlapped-book-cover-with-links.php]: It is related to Figma design. The design has 818px.

File: patterns/hero-overlapped-book-cover-with-links.php (hunk 2 above)
Comment (Arkenon) [reply patterns/hero-overlapped-book-cover-with-links.php]: This content size is also comes from Figma design.

Comment (beafialho) [reply patterns/hero-overlapped-book-cover-with-links.php]: Apologies for the confusion @Arkenon. The designs in Figma serve as the overall vision for the theme, though as they translate to shipping responsive code, some adjustments organically need to happen. For those, please rely on [my comment below](https://github.com/WordPress/twentytwentyfive/pull/309#issuecomment-2356178148).

**Comment by Arkenon** [comment]
@beafialho and @juanfra thanks for your feedbacks.

I have fixed the issues you pointed.

1) Image size setted to 'align:none'
2) Untoggled the `Inner blocks use content width` in the group inside the cover
3) `Regular` spacing preset added to the cover
4) The group inside it setted to wide width instead
5) Minimum height of the cover block removed
6) More spacing addedto spacer block.
7) readme.txt updated

**Comment by beafialho** [comment]
Thank you for your patience @Arkenon! It looks good to me 🚀

**Review by juanfra — APPROVED** [review]
Thank you very much @Arkenon!

**Pull request merged**